ts/sbd-server/src/index.ts

```diff
@@ -57,6 +57,7 @@ export default {
               let opened = 0;
               let active = 0;
               let activeBytesReceived = 0;
+              let recIp = 'no-ip';
```

Comment:
As a general suggestion, you could use the ternary operator for these conditional assignments:
```suggestion
const recIp = 'ip' in item.metadata && typeof item.metadata.ip === 'string'
  ? item.metadata.ip
  : 'no-ip';
```

Reply:
For a brief period I used ternary operators in c, then people started saying not to use them because they were too archaic...